scripts/nuget_pack_apple.sh

```diff
@@ -0,0 +1,388 @@
+
+	log_action 📦 "Installing CocoaPods dependencies for iOS"
+	run_subprocess pushd "${git_root}/apps/ios/src"
+	run_subprocess pod install $verbosity_arg
+	run_subprocess popd
+}
+[[ $prerequisites ]] && do_prerequisites
+
+# Invoke xcodebuild and copy output to nuget folder.
+#
+# Match Azure DevOps xcodebuild invocations by disabling code signing.
+#
+# \param $1 path to Xcode workspace
+# \param $2 scheme to build
+# \param $3 configuration
+# \param $4 sdk
+# \param $5 xcconfig
+# \param $6+ build outputs to copy
+function build_and_copy_output()
+{
+	local path="$1"
+	local scheme="$2"
+	local config="$3"
+	local sdk="$4"
+	local xcconfig="$5"
+	local build_outputs_to_copy=("${@:6}")
+
+	# Set xcodebuild verbosity
+	local verbosity_arg
+	local verbosity_flags=()
+	if [[ $verbosity -le 1 ]]
+	then
+		verbosity_arg="-quiet"
+		# SWIFT_SUPPRESS_WARNINGS=YES conflicts with the -warnings-as-errors option rather
+		# than superseding it, even when paired with SWIFT_WARNINGS_AS_ERRORS=NO, so we
+		# have to live with Swift warning spew in our output for now.  But the flag below
+		# will at least suppress clang warning spew, which is mostly out of our control.
+		verbosity_flags=(GCC_WARN_INHIBIT_ALL_WARNINGS=YES)
+	elif [[ $verbosity -ge 3 ]]
+	then
+		verbosity_arg="-verbose"
+	fi
+
+	# Determine platform/destination for xcodebuild from sdk
+	local platform
+	local config_products_dir="${products_dir}/${config}-${sdk}"
+	case "$sdk"
+	in
+		macosx)
+			platform=macOS;
+			config_products_dir="${products_dir}/${config}";;
+		iphoneos)
+			platform=iOS;;
+		iphonesimulator)
+			platform='iOS Simulator';;
+		*)
+			log_error "Invalid sdk: $sdk";
+			return 1;;
+	esac
+	local destination=generic/platform="$platform"
+
+	# Build
+	log_action 🏗 "Building $platform $config"
+	run_subprocess /usr/bin/xcodebuild \
+		-workspace "$path" \
+		-scheme "$scheme" \
+		-configuration "$config" \
+		-sdk "$sdk" \
+		-destination "$destination" \
+		$verbosity_arg \
+		build \
+		-derivedDataPath "$derived_data_dir" \
+		-xcconfig "$xcconfig" \
+		CODE_SIGNING_ALLOWED=NO \
+		"${verbosity_flags[@]}" \
+	|| return $?
+
+	# Set rsync verbosity
+	if [[ $verbosity -le 1 ]]
+	then
+		verbosity_arg=
+	else
+		verbosity_arg='--verbose'
+	fi
+
+	# Determine destination folder for outputs
+	local nuget_platform_dir="Ship-${sdk}"
+
+	# Copy outputs
+	log_action 📂 "Copying $platform $config output to nuget/$nuget_platform_dir folder"
+	run_subprocess /usr/bin/rsync \
+		--archive \
+		--delete \
+		$verbosity_arg \
+		${build_outputs_to_copy[@]} \
+		"${nuget_dir}/${nuget_platform_dir}/"
+}
+
+# Build for macOS with the specified configuration.
+#
+# \param $1 configuration
+function build_and_copy_macos()
+{
+	build_and_copy_output \
+	"${git_root}/apps/macos/src/FluentTester.xcworkspace" \
+	FluentTester \
+	"$1" \
+	macosx \
+	"${git_root}/.ado/xcconfig/publish_overrides.xcconfig" \
+	${products_dir}/${1}/{"FRN*/libFRN*.a",RCTFocusZone/libRCTFocusZone.a}
```

Comment:
Somewhat concerned we'll forget to add new native modules here in the future

Reply:
Then a good future improvement would be to either invoke the same NuGet packing process that is invoked by the CI task, or  parse the .nuspec file to extract the source and destination info.

However, note that the use of a naming convention helps us a lot here.  If we adopted the name FRNFocusZone/libFRNFocusZone.a for that last library, then the same glob pattern would apply to all of the library outputs, and enforcing that naming convention for new components would ensure we don't have to update anything here.

Reply:
Agreed that RCTFocusZone should be FRNFocusZone, and I would argue win32 should make the same change. 
Hard to make that change here and downstream without breaking something though :')

Reply:
Still, using the FRN* convention for new components will mean we don't have to update this script every time.

Reply:
Yeah, I fully expect every new component to do so. Just no actual CI errors to enforce that. Guess I'll just live with it 🤷🏾